### PR TITLE
test(pie-link): DSW-2229 updated visual tests to split by size

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
## Describe your changes (can list changeset entries if preferable)
- Updated Pie Link tests to split the variations up by size.
- Screenshots should increase from **3** to **6** in number

This is to resolve issues such as:
- Screenshotted variations being cropped due to too large of a screenshot
- Improve screenshot readability and reviewability. 

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have reviewed visual test updates properly before approving

### Reviewer 1 @maledr5 
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 @kevinrodrigues 
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by maledr5 — DISMISSED** [review]
Change looks good. I checked in Percy and shows missing images that I assume are the old ones, but confused on why it shows as missing instead of removed. Is this a known issue to us?

**Comment by xander-marjoram** [comment]
@maledr5 Yep "missing" is just the terminology used. As long as we're happy that only the removed screenshots are missing then that's all good.

File: .changeset/afraid-shoes-cough.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-link": minor
+---
+
+Updated visual tests to split by size
```
Comment (siggerzz) [inline .changeset/afraid-shoes-cough.md:5]: Probably wouldn't create a changeset for this as it result in a package bump that doesnt affect consumers.

**Review by siggerzz — CHANGES_REQUESTED** [review]
Minor comment

**Review by siggerzz — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**